### PR TITLE
JUnit testing files for Datastore Event

### PR DESCRIPTION
**Pull request by Ajide49**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -67,6 +67,46 @@ limitations under the License.
...
 
+    <!-- Test Dependencies
+         <dependency>
+             <groupId>junit</groupId>
+             <artifactId>junit</artifactId>
+             <version>4.12</version>
+             <scope>test</scope>
+         </dependency> -->
+
+         <dependency>
```
Comment (kassidywebb) [inline pom.xml]: Can you fix the formatting on this?

**Comment by kassidywebb** [comment]
Can you explain how this works? It's a lot of new stuff for me, so I'm a little lost going through it!

**Review by kassidywebb — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -67,6 +67,15 @@ limitations under the License.
        <version>1.55.0</version>
     </dependency>
 
+    <!-- https://mvnrepository.com/artifact/com.google.appengine/appengine-testing -->
+<dependency>
```
Comment (ryanluo) [inline pom.xml]: indentation

File: src/main/java/com/google/codeu/data/DatastoreTest.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  } 
+
+//Run this test twice to prove we're not leaking any state across tests.
+ private void doTest() {
+   DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+   assertEquals(0, ds.prepare(new Query("yam")).countEntities(FetchOptions.Builder.withLimit(10)));
```
Comment (ryanluo) [inline src/main/java/com/google/codeu/data/DatastoreTest.java]: What exactly is yam? Shouldn't we test some of our existing Datastore functions?

File: pom.xml (hunk 1 above)
Comment (ryanluo) [reply pom.xml]: +1

File: src/main/java/com/google/codeu/data/DatastoreTest.java [hunk 4]
```diff
@@ -1,6 +1,7 @@
 package com.google.codeu.data;
 
 import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+
```
Comment (ryanluo) [inline src/main/java/com/google/codeu/data/DatastoreTest.java]: nit: remove

**Review by ryanluo — CHANGES_REQUESTED** [review]
I see a lot of commented lines - are they going to land in the final iteration?

File: pom.xml (hunk 1 above)
Comment (Ajide49) [reply pom.xml]: I'll get to it!

File: src/main/java/com/google/codeu/data/DatastoreTest.java (hunk 3 above)
Comment (Ajide49) [reply src/main/java/com/google/codeu/data/DatastoreTest.java]: I will be pushing my actual tests on the Datastore functions later today.

**Comment by ryanluo** [comment]
Ahmed, have you updated this pull request?

**Comment by ryanluo** [comment]
Ahmed can we update this pull request?

**Pull request closed without merging**